src/pkg/services/m365/api/users.go

```diff
@@ -295,6 +297,27 @@ func (c Users) GetInfo(ctx context.Context, userID string) (*UserInfo, error) {
 
 	userInfo.Mailbox = mbxInfo
 
+	if mfs != nil {
+		mf := mfs.GetValue()[0] // we will always have one
+		options := &users.ItemMailFoldersItemMessagesDeltaRequestBuilderGetRequestConfiguration{
+			QueryParameters: &users.ItemMailFoldersItemMessagesDeltaRequestBuilderGetQueryParameters{
+				Top: ptr.To[int32](1), // just one item is enough
+			},
+		}
+		_, err = c.stable.Client().
```

Comment:
hm, looking at this with fresh eyes I'm wondering if we're missing something. This is built on the assumption that the user won't accrue more data in their mailbox during the backup process. Since we have no number to denote how much data is in the mailbox or how full the mailbox is now we're unable to change behavior if we think the mailbox may get full during the backup

It's also unclear if Graph will start returning errors if the mailbox gets full during the backup, even if that happens midway through enumerating items in a folder

Overall, it's hard to say if this will solve _all_ mailbox full failures or just the subset that occur when the mailbox is full prior to launching a backup

To be clear, it does sort of seem like checking up front for fullness simplifies the code (and what's happening on a conceptual level) a bit if we can keep it. However I'm unsure both on the failure modes we get with this approach and what the acceptance criteria for this task is